docs/api/asyncio_con.rst

```diff
@@ -550,3 +571,154 @@ Connection Pools
     .. py:attribute:: free_size
 
         Number of available connections in the pool.
+
+
+.. _edgedb-python-asyncio-api-transaction:
+
+Transactions
+============
+
+The most robust way to make transaction is using a retry loop:
+
+.. code-block:: python
+
+    async for tx in pool.retry():
+        async with tx:
+            await tx.execute("INSERT User { name := 'Don' }")
+
+Note executing queries on ``tx`` object rather than original
+connection or connection pool.
```

Comment:
```suggestion
Note that we execute queries on the ``tx`` object in the above
example, rather than on the original connection pool ``pool`` 
object.
```